psydac/linalg/tests/test_kron_direct_solver.py

```diff
@@ -158,27 +164,28 @@ def compare_solve(seed, comm, npts, pads, periods, direct_solver, transposed=Fal
 
 #===============================================================================
 # tests of the direct solvers
+@pytest.mark.parametrize( 'dtype', [float, complex] )
 @pytest.mark.parametrize( 'seed', [0,2,10] )
 @pytest.mark.parametrize( 'n', [8, 16, 17, 64] )
 @pytest.mark.parametrize( 'p', [1, 3] )
 @pytest.mark.parametrize( 'P', [True, False] )
 @pytest.mark.parametrize( 'nrhs', [1,3] )
-@pytest.mark.parametrize( 'direct_solver', [matrix_to_bandsolver, matrix_to_sparse] )
+@pytest.mark.parametrize( 'direct_solver', [matrix_to_bandsolver] )
```

Comment:
Why have you removed `matrix_to_sparse` from the options?

Reply:
It was an error. I spotted a bug and I was doing some tests.

Reply:
[86b06e9](https://github.com/pyccel/psydac/pull/273/commits/86b06e97fda7a028f2d6ec3f23b6c81d44eaf696)